pkg/parser/epub/metadata.go

```diff
@@ -640,18 +645,38 @@ func (m PubMetadataAdapter) LocalizedSortAs() *manifest.LocalizedString {
 
 func (m PubMetadataAdapter) Accessibility() *manifest.A11y {
 	a11y := manifest.NewA11y()
-	a11y.ConformsTo = m.a11yConformsTo()
-	a11y.Certification = m.a11yCertification()
-	a11y.Summary = m.a11ySummary()
-	a11y.AccessModes = m.a11yAccessModes()
-	a11y.AccessModesSufficient = m.a11yAccessModesSufficient()
-	a11y.Features = m.a11yFeatures()
-	a11y.Hazards = m.a11yHazards()
-	a11y.Exemptions = m.a11yExemptions()
+	ct, refinedA11y := m.a11yConformsTo()
```

Comment:
Would you be able to add test cases for this new logic with the refined metadata? https://github.com/readium/go-toolkit/blob/develop/pkg/parser/epub/metadata_test.go

Reply:
@mickael-menu I've added a new test using that OPF file linked. Let me know if you would like anything else tested

Reply:
That’s fine thanks, feel free to merge the PR